### DISCUSSION TITLE
What's the easiest way to migrate from Postman to Hoppscotch?

### DISCUSSION THREAD

**Discussion by elieobeid7** [hoppscotch/hoppscotch#4542] [General]
I have many teams and workspaces and collections and environments in Postman. Is there a fast way to migrate all of them to Hopscotch?

I don't want to have to export each collection and environment individually from Postman and then import them one by one in Hoppscotch.

**Comment by SanskritiHarmukh** [comment]
Hi there! Thanks for reaching out. Currently, we don’t have an automated functionality to bulk migrate workspaces, collections, and environments from Postman to Hoppscotch. However, we do plan to implement this feature in the future. In the meantime, the only option is to import individual collections and environments into Hoppscotch for migration. We’ll keep you updated as we make progress on this feature.

**Reply by elieobeid7** [reply]
Well noted thanks
